index.html

```diff
@@ -21,83 +22,283 @@
+            <div class="tonic-2">
+              <ul class="tonic-3">
+                <li>CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+                <p class="tonic-para">A daily selection of privately personalized reads;
+                no accounts or sign-ups required.
+                </p>
+            <div class="lag">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+            </div>
+              <div class="btn-2">
+                <button type="button">See Project</button>
+              </div>
+          </div>
+        </div>
+      </li>
+    </ul>
+       
       </section>
-    </main>
+            <!-- About myself page talks of my carrer as sof -->
+
+      <section id="about-page" class="about-page-1">
+        <div class="about-content">
+          <h1 class="about-text">About <br> Myself </h1> 
+          <p class="tonic-para">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me. 
+          </p>
+          <div class="social-media">
+            <h2 class="social-media-text">LET'S CONNECT</h2>
+            <div class="social-media-icons"></div>
+            <ul>
+              <li>
+                <a href="https://github.com/Kossi-stack">
+                  <img
+                    src="assets/github.jpg"
+                    class="icon"
+                    alt="git"/>
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/kossifioklou2406/">
+                  <img
+                    src="assets/Vector.jpg"
+                    class="icon"
+                    alt="linkedin"/>
+                </a>
+              </li>
+              <li>
+                <a href="https://google.com">
+                  <img
+                    src="assets/angellist.jpg"
+                    class="icon"
+                    alt="google"/>
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com">
+                  <img
+                    src="assets/twitter.jpg"
+                    class="icon"
+                    alt="twitter"/>
+                </a>
+              </li>
+              <li>
+                <a href="https://mediamarkt.es">
+                  <img
+                    src="assets/medium.jpg"
+                    class="icon"
+                    alt="mediamarket"/>
+                </a>
+              </li>
+            </ul>
+             <div class="btn-2">
+              <button type="button">Get my resume</button>
+            </div>
+           </div>
+          <div>
+            <h3>Languages</h3>
```

Comment:
- [ ] The languages, Frameworks, and skills part should be `li` tags contained in a `ul` tag. As it is a requirement given:
A screenshot from the project requirements:

![image](https://user-images.githubusercontent.com/68167810/125174878-f4b6cd00-e1d0-11eb-9d43-0dffd58bafd6.png)